linalg/vector.cpp

```diff
@@ -295,14 +292,19 @@ void add(const Vector &v1, double alpha, const Vector &v2, Vector &v)
    {
       const double *v1p = v1.data, *v2p = v2.data;
       double *vp = v.data;
-      int s = v.size;
+      const int s = v.size;
 #ifdef MFEM_USE_OPENMP
       #pragma omp parallel for
-#endif
       for (int i = 0; i < s; i++)
       {
          vp[i] = v1p[i] + alpha*v2p[i];
       }
+#endif
```

Comment:
I think this should be an `#else` and add a matching `#endif` after the `MFEM_FORALL()`?

Reply:
yes your right, Johann